src/main/groovy/com/github/dispader/manifesto/Version.groovy

```diff
@@ -1,18 +1,47 @@
 package com.github.dispader.manifesto
 
 class Version {
+    static String MSG_NO_REPO = "The Manifesto plugin is only useful for projects using git source control."
+    static String MSG_NO_COMMITS = "The Manifesto plugin couldn't find any git commits."
+    static String MSG_NO_TAGS = "The Manifesto plugin couldn't find any version tags. You can create your first tag with `git tag -a v0.1.0`."
 
-    private static getGit() { org.ajoberstar.grgit.Grgit.open() }
+    private static getGit() {
+        try {
+            org.ajoberstar.grgit.Grgit.open()
+        } catch (IllegalArgumentException) {
```

Comment:
Were you hitting an `IllegalArgumentException` when creating `Grgit`?  I wasn't running into that.

This makes me rethink the name of this class.  I think I was a fool when I called this class `Version`.  What you're doing below is really great stuff, and it makes me think that we've actually got something like a "Repository" on our hands, whose constructor can just wrap the Git project and have a method like `versionable`.